packages/@aws-cdk/aws-cloudfront/README.md

```diff
@@ -103,3 +114,118 @@ const distribution = new CloudFrontWebDistribution(this, 'MyDistribution', {
+conjunction with a bucket that is not public to require that your users access your content using CloudFront URLs and not S3 URLs directly.
+
+### Domain Names and Certificates
+
+When you create a distribution, CloudFront returns a domain name for the distribution, for example: `d111111abcdef8.cloudfront.net`. CloudFront
+distributions use a default certificate (`*.cloudfront.net`) to support HTTPS by default. If you want to use your own domain name, such as
+`www.example.com`, you must associate a certificate with your distribution that contains your domain name. The certificate must be present in the AWS
+Certificate Manager (ACM) service in the US East (N. Virginia) region; the certificate may either be created by ACM, or created elsewhere and
+imported into ACM.
+
+```ts
+const myCertificate = new acm.DnsValidatedCertificate(this, 'mySiteCert', {
+  domainName: 'www.example.com',
+  hostedZone,
+});
+new cloudfront.Distribution(this, 'myDist', {
+  defaultBehavior: { origin: cloudfront.Origin.fromBucket(myBucket) },
+  certificate: myCertificate,
+});
+```
+
+### Multiple Behaviors & Origins
+
+Each distribution has a default behavior which applies to all requests to that distribution; additional behaviors may be specified for a
+given URL path pattern. Behaviors allow routing with multiple origins, controlling which HTTP methods to support, whether to require users to
+use HTTPS, and what query strings or cookies to forward to your origin, among others.
+
+The properties of the default behavior can be adjusted as part of the distribution creation. The following example shows configuring the HTTP
+methods and viewer protocol policy of the cache.
+
+```ts
+const myWebDistribution = new cloudfront.Distribution(this, 'myDist', {
+  defaultBehavior: {
+    origin: cloudfront.Origin.fromBucket(myBucket),
+    allowedMethods: AllowedMethods.ALLOW_ALL,
+    viewerProtocolPolicy: ViewerProtocolPolicy.REDIRECT_TO_HTTPS,
+  }
+});
+```
+
+Additional cache behaviors can be specified at creation, or added after the initial creation. Each additional behavior is associated with an origin,
+and enable customization for a specific set of resources based on a URL path pattern. For example, we can add a behavior to `myWebDistribution` to
+override the default time-to-live (TTL) for all of the images.
+
+```ts
+myWebDistribution.addBehavior('/images/*.jpg', {
+  origin: cloudfront.Origin.fromBucket(myOtherBucket),
+  viewerProtocolPolicy: ViewerProtocolPolicy.REDIRECT_TO_HTTPS,
+  defaultTtl: cdk.Duration.days(7),
+});
+```
+
+These behaviors can also be specified at distribution creation time.
+
+```ts
+const bucketOrigin = cloudfront.Origin.fromBucket(myBucket);
+new cloudfront.Distribution(this, 'myDist', {
+  defaultBehavior: {
+    origin: bucketOrigin,
+    allowedMethods: AllowedMethods.ALLOW_ALL,
+    viewerProtocolPolicy: ViewerProtocolPolicy.REDIRECT_TO_HTTPS,
+  },
+  additionalBehaviors: [{
+    '/images/*.jpg', {
```

Comment:
I am assuming this should be ":" and not ","

Reply:
Correct.